src/tests/teamcity-event-listener.tests.csproj

```diff
@@ -28,6 +29,7 @@
     <DefineConstants>TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <LangVersion>3</LangVersion>
```

Comment:
I can remove it, but I had several cases with broken CI builds when I used some new features of C# and local builds worked fine. Version was changed to C# 5.0